pandas/core/generic.py

```diff
@@ -2492,17 +2492,25 @@ def to_xarray(self):
         """
         Return an xarray object from the pandas object.
 
+        Return the xarray equivalent of the pandas object. `xarray
+        <http://xarray.pydata.org/en/stable/>`__ is a
+        Python package that allows to handle N-dimensional data.
+
```

Comment:
I'm not sure if we need to mention this here.

Reply:
I was trying to extend the previous short summary. Do you suggest to simply remove it or is there something else we should mention here?

Reply:
I think we can remove it.  We describe `xarray` elsewhere in the docs anyhow.